pages/admin/accounts/[id]/manage-forms.tsx

```diff
@@ -0,0 +1,237 @@
+                  {titleEn} / {titleFr}
+                </h2>
+
+                <span
+                  className="block rounded px-2 py-1 "
+                  style={{
+                    backgroundColor: backgroundColor,
+                    border: `2px solid ${borderColor}`,
+                  }}
+                >
+                  {isPublished ? t("published") : t("draft")}
+                </span>
+              </div>
+
+              {overdue[id] && <OverdueStatus level={overdue[id].level} />}
+
+              {/* linking to existing page for now */}
+              <div className="mt-10 flex flex-row items-end justify-between">
+                <div>
+                  <LinkButton.Secondary
+                    href={`/${i18n.language}/form-builder/settings/${id}/form?backLink=${formUser.id}`}
+                    className="mb-2 mr-3"
+                  >
+                    {t("manageOwnerships")}
+                  </LinkButton.Secondary>
+                  <LinkButton.Secondary
+                    href={`/${i18n.language}/form-builder/responses/${id}`}
+                    className="mb-2 mr-3"
+                  >
+                    {t("gotoResponses")}
+                  </LinkButton.Secondary>
+                </div>
+                <div>
+                  {canManageForms && (
+                    <Dropdown>
+                      <DropdownMenuPrimitive.Item
+                        className={`${themes.destructive} ${themes.base} !block !cursor-pointer`}
+                        onClick={() => {
+                          activeForm.current = { id, isPublished };
+                          setShowConfirm(true);
+                        }}
+                      >
+                        {t("deleteForm")}
+                      </DropdownMenuPrimitive.Item>
+                    </Dropdown>
+                  )}
+                </div>
+              </div>
+            </li>
+          );
+        })}
+      </ul>
+      <ConfirmDelete
+        onDeleted={() => {
+          setShowConfirm(false);
+          refreshData();
+        }}
+        show={showConfirm}
+        id={activeForm.current?.id || ""}
+        isPublished={activeForm.current?.isPublished || false}
+        handleClose={setShowConfirm}
+      />
+    </>
+  );
+};
+
+const BackToAccounts = () => {
+  const { t } = useTranslation("admin-users");
+  return <BackLink href="/admin/accounts">{t("backToAccounts")}</BackLink>;
+};
+
+ManageForms.getLayout = (page: ReactElement) => {
+  return (
+    <AdminNavLayout user={page.props.user} backLink={<BackToAccounts />}>
+      <TemplateStoreProvider {...{ locale: page.props.locale }}>{page}</TemplateStoreProvider>
+    </AdminNavLayout>
+  );
+};
+
+export const getServerSideProps = requireAuthentication(
+  async ({ locale, params, user: { ability } }) => {
+    const id = params?.id || null;
+    checkPrivileges(ability, [{ action: "view", subject: "Setting" }]);
```

Comment:
To scope it down to the context of what's being retrieved in terms of information we could instead use:

```js
    checkPrivileges(ability, [
      { action: "view", subject: "User" },
      {
        action: "view",
        subject: {
          type: "FormRecord",
          // Passing an empty object here just to force CASL evaluate the condition part of a permission.
          // Will only allow users who have privilege of Manage All Forms
          object: {},
        },
      },
    ]);
```